code/modules/mob/living/carbon/carbon.dm

```diff
@@ -269,14 +269,22 @@ mob/living
 						)
 			// BEGIN HUGCODE - N3X
 			else
-				if (istype(src,/mob/living/carbon/human) && src:w_uniform)
-					var/mob/living/carbon/human/H = src
-					H.w_uniform.add_fingerprint(M)
-				playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
-				M.visible_message( \
-					"\blue [M] gives [src] a [pick("hug","warm embrace")].", \
-					"\blue You hug [src].", \
+				if(M.zone_sel.selecting == "head")
+					playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+					M.visible_message( \
+					"<span class='notice'> [M] pats [src] on the head.</span>", \
+					"<span class='notice'> You pat [src] on the head.</span>", \
 					)
+				else
+					playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
```

Comment:
The playsound here and and under the `if` statement can just be moved to before the `if` statement.